docs/swagger/xrpl-routes.yml

```diff
@@ -0,0 +1,65 @@
+paths:
```

Comment:
there should not be any chain-specific routes, as this was the main change introduced in https://github.com/hummingbot/gateway/pull/147.

pls refactor these routes to use the existing endpoints where possible. if there is new functionality that needs to be introduced via a separate endpoint, it should be generalized and not specific to Ripple

Reply:
Thanks, this was artifact from version 1, I have removed it.